boards/arm/v2m_musca/v2m_musca_defconfig

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_ARM=y
+CONFIG_SOC_SERIES_MUSCA=y
+CONFIG_SOC_V2M_MUSCA_A=y
+CONFIG_BOARD_MUSCA_A=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_RUNTIME_NMI=y
+
```

Comment:
No MPU support, @galak @agross-linaro ?

Reply:
You should also enable TrustZone-M: 
```
# Enable TrustZone-M
CONFIG_ARM_TRUSTZONE_M=y

```
So the user has option to use this board with or without support for TrustZone-M.

Reply:
added this in